forui/test/src/widgets/accordion/accordion_golden_test.dart

```diff
@@ -0,0 +1,78 @@
+@Tags(['golden'])
+library;
+
+import 'package:flutter/material.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:forui/forui.dart';
+import '../../test_scaffold.dart';
+
+void main() {
```

Comment:
We can use `TestScaffold.app` instead, which wraps the test scaffold in a MaterialApp.